internal/db_clean_test.go

```diff
@@ -0,0 +1,69 @@
+package internal_test
```

Comment:
I don't understand why you use the `internal` pkg here. My understanding of `internal` pkgs is that only the direct parent pkg can use them (https://golang.org/cmd/go/#hdr-Internal_Directories). So here, only code at the root pkg of `fabric8-common` could use it. So, in practice, who is going to be able to use that code exactly ?

Reply:
This package is to be used by common itself. This is why I put it to internal.

Reply:
ok